app/src/main/res/drawable/outline_photo_camera_24.xml

```diff
@@ -0,0 +1,5 @@
+<vector android:height="25dp" android:tint="#000000"
```

Comment:
I will fix the name however the icons I needed to make the design weren't in the default Icon Pack but were available in the extended. But the extended pack is quite large so I didn't think it would be worth it to add it as we will only use a few Icons